css/sekcje/howitworks.css

```diff
@@ -0,0 +1,74 @@
+/* ------------------------------MOBILE */
+
+.howitworks {
+    padding-top: 80px;
+    margin: 0 auto;
+}
+
+.howitworks-title {
+    margin-bottom: 14px;
+}
+.howitworks-text {
+    text-align: center;
+    margin-bottom: 44px;
+}
+
+.howitworks-fieldset {
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+    margin: 0 auto;
+    align-items: center;
+}
+
+.howitworks-field {
+    width: 335px;
+    height: 112px;
+    border-radius: 20px;
+    background-color: var(--clr-white);
+    display: flex;
+    align-items: center;
+    padding: 20px;
+}
+
+.howitworks-steps-btn {
+    background-color: var(--clr-red);
+    width: 136px;
+    height: 48px;
+    border-radius: 200px;
+    border: none;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 1.33333;
+    letter-spacing: -0.04em;
+    color: var(--clr-white);
+    margin: 12px 0;
+    margin-right: 14px;
+    display: inline;
+}
+
+.green {
+    background-color: var(--clr-green);
+}
+
+.yellow {
+    background-color: var(--clr-yellow);
+}
+
```

Comment:
Skoro zrobiłaś green i yellow to zrób też red i usuń właściwośc background-color z steps-btn.

Reply:
ofkors